examples/go.mod

```diff
@@ -3,138 +3,191 @@ module github.com/pulumi/pulumi-aws/examples/v6
 go 1.18
 
 require (
-	github.com/aws/aws-sdk-go v1.38.35
-	github.com/pulumi/pulumi/pkg/v3 v3.31.1
-	github.com/stretchr/testify v1.7.1
+	github.com/aws/aws-sdk-go v1.44.122
+	github.com/pulumi/pulumi/pkg/v3 v3.71.0
```

Comment:
This implicitly upgrades ProgramTest framework hitting https://github.com/pulumi/pulumi/issues/12859 ; I think this explains TestAccWebServer failure.